sonaranalyzer-dotnet/src/SonarAnalyzer.Vsix/SonarAnalyzer.Vsix.csproj

```diff
@@ -96,12 +96,16 @@
       <Name>SonarAnalyzer.VisualBasic</Name>
       <IncludeOutputGroupsInVSIX>BuiltProjectOutputGroup</IncludeOutputGroupsInVSIX>
       <IncludeOutputGroupsInVSIXLocalOnly>DebugSymbolsProjectOutputGroup</IncludeOutputGroupsInVSIXLocalOnly>
+      <!-- https://github.com/dotnet/sdk/issues/433 -->
+      <AdditionalProperties>TargetFramework=net461</AdditionalProperties>

```

Comment:
As described in the linked issue, multi-targeting .net core projects cannot be referenced by VSIX projects. I order to work around this we have to filter the target framework by using **AdditionalProperties**